remote-monitoring/docker-compose.dotnet.yml

```diff
@@ -12,7 +12,7 @@ services:
       - iothubmanager
       - devicesimulation
       - telemetry
-      - uiconfig
+      - config
```

Comment:
Do you also want to remove the circular dependency bug with this change?

Reply:
👍

Reply:
done